src/modules/dashboard/sagas/colonyCreate.ts

```diff
@@ -194,34 +189,33 @@ function* colonyCreate({
       });
     }
 
-    yield createGroupedTransaction(deployOldRoles, {
+    yield createGroupedTransaction(setAdministrationRole, {
       context: COLONY_CONTEXT,
-      methodName: 'addExtension',
-      params: { contractName: 'OldRoles' },
-      ready: false,
+      methodName: 'setAdministrationRole',
+      params: {
+        address: walletAddress,
```

Comment:
This shouldn't be set here for this one, or for `setFundingRole`, they should be set to the `OneTxPayment` extension address below.